build.wake

```diff
@@ -13,100 +13,103 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package build_wake
 from wake import _
 
 # Useful build variants (arguments to all)
 def toVariant = match _
     "default" = Pass (Pair "native-cpp14-release" "native-c11-release")
-    "static"  = Pass (Pair "native-cpp14-static"  "native-c11-static")
-    "debug"   = Pass (Pair "native-cpp14-debug"   "native-c11-debug")
-    "wasm"    = Pass (Pair "wasm-cpp14-release"   "wasm-c11-release")
+    "static" = Pass (Pair "native-cpp14-static" "native-c11-static")
+    "debug" = Pass (Pair "native-cpp14-debug" "native-c11-debug")
+    "wasm" = Pass (Pair "wasm-cpp14-release" "wasm-c11-release")
     s = Fail "Unknown build target ({s})".makeError
 
+
 export def build: List String => Result String Error = match _
     "tarball", Nil = tarball Unit | rmap (\_ "TARBALL")
     kind, Nil =
         require Pass variant = toVariant kind
         all variant | rmap (\_ "BUILD")
     _ = Fail "no target specified (try: build default/debug/tarball)".makeError
 
+
 export def install: List String => Result String Error = match _
     dest, kind, Nil = doInstall (in cwd dest) kind | rmap (\_ "INSTALL")
     dest, Nil = doInstall (in cwd dest) "default" | rmap (\_ "INSTALL")
     _ = Fail "no directory specified (try: install /opt/local)".makeError
 
+
 # Build all wake targets
 def targets =
-    buildWake, buildWakeBox, buildFuseDaemon, buildShim, buildBSP, buildLSP, buildJobCache, buildWakeFormat, Nil
+    buildWake,
+    buildWakeBox,
+    buildFuseDaemon,
+    buildShim,
+    buildBSP,
+    buildLSP,
+    buildJobCache,
+    buildWakeFormat,
+    Nil
+
 
 def all variant =
-    require Pass x =
-        map (_ variant) targets
-        | findFail
+    require Pass x = map (_ variant) targets | findFail
     Pass (flatten x)
 
+
 # Install wake into a target location
 def doInstall dest kind =
     require Pass variant = toVariant kind
-    require Pass datfiles =
-        sources "{here}/share" `.*`
+    require Pass datfiles = sources "{@here}/share" `.*`
     def binfilesResult = all variant
     def releaseBin exe = installAs "{dest}/{replace `\.[^.]*$` '' exe.getPathName}" exe
-    require Pass datinstall =
-        findFailFn (installIn dest ".") datfiles
-    require Pass readmeSource =
-        source "README.md"
-    require Pass readme =
-        installIn "{dest}/share/doc/wake" "." readmeSource
+    require Pass datinstall = findFailFn (installIn dest ".") datfiles
+    require Pass readmeSource = source "README.md"
+    require Pass readme = installIn "{dest}/share/doc/wake" "." readmeSource
     require Pass binfiles = binfilesResult
-    require Pass bininstall =
-        findFailFn releaseBin binfiles
+    require Pass bininstall = findFailFn releaseBin binfiles
```

Comment:
So, our style guide previously recommended that every `def/require` body starts on a new line.
I think that was too annoying, but I do think there are two cases we should use for format:

One-line style / collection, line up the `=`:
```
def abc = 345
def d    = 4546
def ef   = 123
```

Multi-line style, don't line up the `=`, but indent every body
```
def abc =
    ...
def d =
    ...
def ef =
    ... one of these is really long, which is why they all got dropped
```

Reply:
As a motivating example to do the work in the formatter: in scala/chisel it ended up really important to readability that "ready" and "valid" are both 5 characters long. When we switched to a third signal, that heavily motivated the choice to use "offer", "abort", and "ready". If you don't do the line up in the formatter, people will pick variable names to make things line up, sometimes impacting understandability by bad name choices.

Reply:
Jake and I are going to explore options here based on discussions in the meeting and come back with more info later